solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
+    'concrete': {
+        'seller': 'mozilla-concrete',
+        'products': [
+            {
+                'name': 'brick',
+                'amount': Decimal('10')
```

Comment:
aren't these going to need more definition like recurring subscription fee and period of validity? I guess we could amend it later.